components/admin_console/admin_navbar_dropdown/admin_navbar_dropdown.tsx

```diff
@@ -65,12 +65,17 @@ class AdminNavbarDropdown extends React.Component {
                             id='select_team.icon'
                             defaultMessage='Select Team Icon'
                         >
-                            {(title) => (
-                                <i
-                                    className='fa fa-exchange'
-                                    title={title}
-                                />
-                            )}
+                            {(title) => {
+                                if (typeof title === 'string') {
```

Comment:
Is the intent to change the behaviour so that there's no `<i>` element if there's no `title` parameter?

Reply:
@mkraft  ah. title can be string or React.Node .    I changed it so title will be returned. Line 77

Reply:
@mkraft   title can be string or React.Node.  I changed it so that it returns the Node. line 77.  good catch